rxandroidble/src/main/java/com/polidea/rxandroidble2/internal/util/RxBleAdapterWrapper.java

```diff
@@ -27,6 +27,9 @@ public RxBleAdapterWrapper(@Nullable BluetoothAdapter bluetoothAdapter) {
     }
 
     public BluetoothDevice getRemoteDevice(String macAddress) {
+        if (bluetoothAdapter == null) {
```

Comment:
I am afraid that most of these changes in `RxBleAdapterWrapper` would cause a lot of confusion if adapter would be null. Maybe throwing a generic `BleException` with an appropriate message would be better idea?

Reply:
Ok